src/Compilers/CSharp/Test/Emit2/Semantics/ExtensionTypeTests.cs

```diff
@@ -821,9 +821,6 @@ class UnderlyingClass : I
             // (11,12): error CS0541: 'R1.M()': explicit interface declaration can only be declared in a class, record, struct or interface
             //     void I.M() { }
             Diagnostic(ErrorCode.ERR_ExplicitInterfaceImplementationInNonClassOrStruct, "M").WithArguments("R1.M()").WithLocation(11, 12),
-            // (13,45): error CS9307: A base extension must be an extension type.
-            // explicit extension R2 for UnderlyingClass : I
```

Comment:
>// explicit extension R2 for UnderlyingClass : I [](http://example.com/codeflow?start=12&length=48)

I think this should still be be an error. Perhaps, a general one saying base extensions aren't supported yet, or a parsing error.

Reply:
We have a ERR_NotYetImplementedInRoslyn error that can be used for this case